### PR TITLE
Make Skynet running correctly again

### PR DESCRIPTION
**Pull request by RndName**
Updated small things to get skynet running again as it should:

- All AWACs flights are now added correctly
- Point defence SAMs will be added
- Long Range SAMs are now operating as EWR as recommended by the manual
- adopt to changes from https://github.com/dcs-liberation/dcs_liberation/pull/1300

first part for #324 

this changes are meant to work on top of https://github.com/dcs-liberation/dcs_liberation/pull/1300 as they use the naming convention of the SAM Sites which i implemented there.

### CODE REVIEW

**Comment by RndName** [comment]
#645

File: game/operation/operation.py [hunk 1]
```diff
@@ -485,14 +484,15 @@ def generate_lua(
...
-                    luaData["TargetPoints"][flightTargetName] = {
+                    luaData["TargetPoints"][flight_count] = {
                         "name": flightTargetName,
                         "type": flightTargetType,
                         "position": {
                             "x": flightTarget.position.x,
                             "y": flightTarget.position.y,
                         },
                     }
+                    i += 1
```
Comment (DanAlbert) [inline game/operation/operation.py]: Am I reading this right? It looks like this is the `i` from the JTAC loop on L458, and that it does nothing. It looks like you wanted to increment `flight_count`?

**Review by DanAlbert — COMMENTED** [review]
(I see it's a draft but I took a quick look anyway)

File: game/operation/operation.py (hunk 1 above)
Comment (RndName) [reply game/operation/operation.py]: Good catch! THANK You!
I have not tested this one 100% yet, thats why i left it as draft :)

**Pull request closed without merging**